radix-engine-interface/src/types/node_layout.rs

```diff
@@ -26,8 +26,7 @@ pub enum SysModuleId {
     Metadata,
     Royalty,
     AccessRules,
-    Object,
-    Virtualized,
+    User,
```

Comment:
This has changed name 3 times now (Self => Object => User) - I'd quite like an agreement on the name of this before merging this PR / integrating this into the node :)

EG I think I'd probably prefer the following from most-to-least "Main" / "Principal" / "Default" / "Primary" / "Standard" / "Normal" / "Client" / "User"

Reply:
It's changed because abstraction of this module has changed over time. The current abstraction as an enum may also not be the correct one as this is more of a number.

I prefer "User"/"Application" as it refers to User/Application Memory Space and in this case it would define the "start" of the user space.

Reply:
I think I don't understand the distinction between ObjectModuleId and SysModuleId and what the abstractions represent. It seems 1:1 with ObjectModuleId at the moment.

(and FWIW I prefer "Application" to "User" because in docs I've used "User" (Transaction) to mean a user of the network, whereas here, it means "not-system-controlled" I guess?)